doc/github_pages/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Eclipse OpenBSW Documentation</title>
+  </head>
+  <body>
+    <h1>Eclipse OpenBSW Documentation</h1>
+    <h2><a href="sphinx_docs/doc/index.html">Welcome to Eclipse OpenBSW</a></h2>
+    <h3><a href="doxygenOut/html/index.html">Doxygen-generated API docs</a></h3>
+    <h3>Coverage reports</h3>
+    <ul>
+        <li><a href="code_coverage_url">Code coverage report</a></li>
+        <li><a href="api_doc_coverage_url">API doc coverage report</a></li>
+    </ul>
+  </body>
+</html>
```

Comment:
Missing /n

Reply:
Have added a newline at the end of `doc/github_pages/index.html`